src/prefect/server/api/ui/schemas.py

```diff
@@ -28,7 +28,8 @@ async def validate_obj(
     json_schema: dict[str, Any] = Body(
         ...,
         embed=True,
-        alias="schema",
+        validation_alias="schema",
+        serialization_alias="schema",
```

Comment:
This is whack, but I tested this locally and it seems to do everything that we need it to.
```suggestion
        alias="schema",
        validation_alias="schema",
```